src/fonts.php

```diff
@@ -19,6 +19,10 @@ function __construct() {
 		}
 
 		public function gather_google_fonts( $block_content, $block ) {
+			if ( $block_content === null ) {
+				return $block_content;
+			}
+
```

Comment:
Please remove, the block_content isn't actually used here, might cause fonts to stop loading